ocaml/xapi/xapi_crashdump.ml

```diff
@@ -11,22 +11,15 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-exception Not_implemented
-
-let nothrow f () = try f() with _ -> ()
-
-let create ~__context ~vM ~vDI =
-  let cdumpref = Ref.make() in
-  let uuid = Uuid.to_string (Uuid.make_uuid()) in
-  Db.Crashdump.create ~__context ~ref:cdumpref ~uuid ~vM ~vDI ~other_config:[];
-  cdumpref
 
 let destroy ~__context ~self =
```

Comment:
This is a XenAPI call, crashdump.destroy: https://xapi-project.github.io/xen-api/classes/crashdump.html,
so I think it's best to stick to the existing conventions and keep this file.